clang/lib/3C/ConstraintBuilder.cpp

```diff
@@ -68,6 +70,48 @@ void processRecordDecl(RecordDecl *Declaration, ProgramInfo &Info,
   }
 }
 
+void processVarDecl(VarDecl *VD, ProgramInfo &Info,
+                    ASTContext *Context, ConstraintResolver CB) {
+  if(lastRecord2 != nullptr) {
```

Comment:
Add comment here to say what your code is doing. It's a little mysterious why you have this conditional.